api_layer/source/controllers/Matches.js

```diff
@@ -7,4 +7,23 @@ module.exports = (api) => {
                 .then(list => res.json(list))
                 .catch(err => res.json(err));
         });
+
+    /**
+     * @api {get} /Match/:gameToken Request matches with given gameToken
+     * @apiName GetMatches
+     * @apiGroup Matches
+     * @apiDescription
+     * Returns a list of all matches that contain the provided gameToken.
+     * This list is empty if no matches were found.
+     *
+     * @apiParam {String} gameToken The unique game token of a game.
+     * @apiSuccess (200) Success
+     *
+     */
+    api.route('/Matches/:gameToken')
```

Comment:
By convention, we are apprentantly have routes as lowercase. i.e /matches/:gameToken